tests/test_web_urldispatcher.py

```diff
@@ -0,0 +1,103 @@
+import pytest
+import tempfile
+import aiohttp
+from aiohttp import web
+import os
+import shutil
+import asyncio
+
+SERVER_HOST = '127.0.0.1'
+SERVER_PORT = 8080
```

Comment:
Please use `unused_port` fixture here.
See https://github.com/KeepSafe/aiohttp/blob/90f16168fd8510fe930994c4b4170a18cd8f2568/tests/test_run_app.py#L35-L36 for inspiration.